File: expresslk-services/ExpressLK-core/src/main/java/lk/express/SearchCriteria.java
package lk.express;

import javax.xml.bind.annotation.XmlRootElement;
import javax.xml.bind.annotation.XmlType;

@XmlRootElement
@XmlType(name = "SearchCriteria", namespace = "http://express.lk")
public class SearchCriteria extends BaseCriteria {

	private static final long serialVersionUID = 1L;

	private int fromCityId;
	private String fromCity;
	private int toCityId;
	private String toCity;
	private boolean roundTrip;
	private LegCriteria[] legCriterion;

	public SearchCriteria() {
		legCriterion = new LegCriteria[2];
	}

	public int getFromCityId() {
		return fromCityId;
	}

	public void setFromCityId(int fromCityId) {
		this.fromCityId = fromCityId;
	}

	public String getFromCity() {
		return fromCity;
	}

	public void setFromCity(String fromCity) {
		this.fromCity = fromCity;
	}

	public int getToCityId() {
		return toCityId;
	}

	public void setToCityId(int toCityId) {
		this.toCityId = toCityId;
	}

	public String getToCity() {
		return toCity;
	}

	public void setToCity(String toCity) {
		this.toCity = toCity;
	}

	public boolean isRoundTrip() {
		return roundTrip;
	}

	public void setRoundTrip(boolean roundTrip) {
		this.roundTrip = roundTrip;
	}

	public LegCriteria[] getLegCriterion() {
		return legCriterion;
	}

	public void setLegCriterion(LegCriteria[] legCriterion) {
		this.legCriterion = legCriterion;
	}

	@Override
	public String toString() {
		return super.toString();
	}
}

File: expresslk-webportal/module/Api/src/Api/Client/Soap/Core/SearchCriteria.php
<?php

namespace Api\Client\Soap\Core;

include_once('BaseCriteria.php');

class SearchCriteria extends BaseCriteria
{

    /**
     * @var string $fromCity
     * @access public
     */
    public $fromCity = null;

    /**
     * @var int $fromCityId
     * @access public
     */
    public $fromCityId = null;

    /**
     * @var LegCriteria[] $legCriterion
     * @access public
     */
    public $legCriterion = null;

    /**
     * @var boolean $roundTrip
     * @access public
     */
    public $roundTrip = null;

    /**
     * @var string $toCity
     * @access public
     */
    public $toCity = null;

    /**
     * @var int $toCityId
     * @access public
     */
    public $toCityId = null;

    /**
     * @param int $clientId
     * @param string $discountCode
     * @param string $ip
     * @param string $source
     * @param string $fromCity
     * @param int $fromCityId
     * @param boolean $roundTrip
     * @param string $toCity
     * @param int $toCityId
     * @access public
     */
    public function __construct($clientId, $discountCode, $ip, $source, $fromCity, $fromCityId, $roundTrip, $toCity, $toCityId)
    {
      parent::__construct($clientId, $discountCode, $ip, $source);
      $this->fromCity = $fromCity;
      $this->fromCityId = $fromCityId;
      $this->roundTrip = $roundTrip;
      $this->toCity = $toCity;
      $this->toCityId = $toCityId;
    }

}
